test/metabase/api/notification_test.clj

```diff
@@ -342,7 +386,7 @@
                                             (mt/user-http-request user-or-id :put expected-status (format "notification/%d" (:id notification))
                                                                   (assoc notification :updated_at (t/offset-date-time))))
               change-notification-creator (fn [user-id]
-                                            (t2/update! :model/Notification (:id notification) {:creator_id user-id}))
+                                            (t2/update! :notification (:id notification) {:creator_id user-id}))
```

Comment:
Can you add a comment for why the model hooks are being bypassed here?

Reply:
yea will do, but the idea here is we prevent updating creator in the before-update hook, so in tests we need to bypass it by using table name.